### ISSUE TITLE
Usage with SSR (getServerSideProps) instead of SSG?

### ISSUE THREAD

**Issue by noahflk** [QuiiBz/next-international#23]
I can't use `getStaticProps` because I rely on `getServerSideProps` on some pages. How can I make my translations available? Is there an equivalent to `getLocaleStaticProps()` for SSR instead of SSG?

**Comment by QuiiBz** [comment]
That's a good point.

For now, you can load locales client-side without any `getStaticProps` nor `getServerSideProps` (see https://github.com/QuiiBz/next-international#load-initial-locales-client-side). Let me see how we can support `getServerSideProps`, but it should be similar to `getStaticProps`.

**Comment by noahflk** [comment]
I think it's mainly a TypeScript issue. If I suppress errors it works just fine:

```
// @ts-ignore
export const getServerSideProps: GetServerSideProps = getLocaleStaticProps((ctx) => {
  // @ts-ignore
  return protectedRoute(ctx);
});
```

**Comment by QuiiBz** [comment]
Just released version `0.2.0` which includes #25 and #24, feel free to reopen if the issue isn't fixed!
